src/fixes.rs

```diff
@@ -477,6 +485,7 @@ pub fn merge_overlapping_fixes(
                 end: TextWidth::from_str(&file_content).as_offset(),
             },
             suggestion: file_content_after.to_string(),
+            description: String::new(),
```

Comment:
This one will appear when we get any overlapping diagnostics. I would put something like "Fix whole file" here, or something other. I think we would have to make a separate function which would return non resolved nested fixes. It's better to be confronted with @Draggu , as he's the one that requested this feature